### ISSUE TITLE
Error - Excel cannot openthe file... file format/file extension is not valid. 

### ISSUE THREAD

**Issue by douglasshepherd** [LG-Experiment/Scripts#4]
I will look into my AV software and Trusted Sources to see if something is blocking access on my Win11Pro machine.  I am attaching a screenshot.  I renamed the file after I received the error a few times, which is why the name is a little different in the screenshot.

![Screenshot_20230222_100404](https://user-images.githubusercontent.com/19764058/220717924-699fa643-dd81-4098-a946-a7291b102cdb.png)

**Comment by douglasshepherd** [comment]
I reviewed my Trust Center settings and AV and all seems fine there.  I have also attempted this in Office 365 Business Online and receive the same warning and inability to open the file.  Using MS Excel - Microsoft 365 MSO (Version 2301 Build 16.0.16024.20002) 64-bit.  I am updating now to see if that resolves error.

**Comment by douglasshepherd** [comment]
Issue resolved. I originally direct downloaded the files and something bizarre seems to have happened in the download process. After a bit of fiddling, I changed the file extension to .xls and was then able to open the Excel file.  Oddly, however, it piped the contents of the README file to the excel spreadsheet.  I cloned the repo via GitHub for Desktop and it seems fine now. Let's just file that under user error, shall we?
